.ci_support/migrations/cuda118.yaml

```
@@ -0,0 +1,99 @@
+migrator_ts: 1692828152
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+  paused: false
```

Also think we need this so that re-rendering uses our migrator. Guessing this is what was missing before

```suggestion
  use_local: true
  paused: false
```